JenkinsfileStressTests.groovy

```diff
@@ -87,7 +87,7 @@ pipeline {
                                     break
                                 case "s3-local":
                                     sh 'docker run -d --name=cassandra -p 9042:9042 cassandra:3.11.10'
-                                    sh 'docker run -d --name=elasticsearch -p 9200:9200 --env "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:6.3.2'
+                                    sh 'docker run -d --name=elasticsearch -p 9200:9200 --env "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:7.10.2'
```

Comment:
That's just `s3-local` case... what about the others? You don't use the provisioning btw?